doc/guides/coverage.rst

```diff
@@ -141,6 +141,13 @@ or::
 
 which will produce ``sanity-out/coverage/index.html`` with the report.
 
+Note that the sanitycheck coverage reports are currently created with
+`lcov`_, with a minimum version of 1.14. Some Linux distributions may
```

Comment:
A bit of a silly comment: the OS provided lcov should just work when generating reports for the POSIX arch (as normally one would also use the  OS provided gcc/gcov). So 1.13 is just fine for native_posix.

Reply:
So is 1.13 sufficient (and we don't really need 1.14)?

Reply:
you need 1.14 in order to gather coverage data from QEMU targets, which are the only way to get coverage data for some subsystems like user mode